settings.py

```diff
@@ -82,3 +82,7 @@
 PR_STALE_HOURS = 36
 
 API_COOLDOWN_RESET_PADDING = 30
+
+# list of autocracy members
+with open("data/autocracy.txt", "r") as f:
```

Comment:
It's going to get removed in the next iteration of the PR anyway (where it's a meritocracy).